### PR TITLE
Prepare to release optimus v1

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose
_Please explain the purpose of this PR and include links to any GitHub issues that it fixes:_

- No issue is linked to this PR.
---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- No changes.
---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

- No instructions.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] Let Jodi (**jodihir**) know when there is user-facing documentation updates needed!
- [ ] This PR applied the Mint style guide for WDL.
- [ ] This PR has no WDL linting errors reported by [miniwdl](https://github.com/chanzuckerberg/miniwdl)
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: pipelines/optimus/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Optimus Pipeline
+
+The Optimus pipeline, developed by the Data Coordination Platform of the Human Cell Atlas (HCA DCP), processes 3' prime single cell gene expression data from the 10X Genomics v2 (and v3)assay. 
+
+Optimus is a quality control, alignment and transcriptome quantification module. It corrects Cell Barcodes (CBs) and Unique Molecular Identifiers (UMIs), aligns reads to the genome, generates an expression count matrix in a UMI-aware manner, detects empty droplets, calculates summary statistics, and returns outputs in BAM and Zarr file formats. Special care is taken to avoid the removal of reads that are not aligned or that do not contain recognizable barcodes. This design (which differs from many pipelines currently available) allows the use of the entire dataset by those who may want to use alternative filtering or leverage the data for methodological development associated with the data processing.
+
+More details can be found in the [Optimus Acceptance Report](#)
```
Comment (rexwangcc) [inline pipelines/optimus/README.md]: I'm not sure if the report(in the Google Docs form) is publicly available (or should be publicly available). I personally prefer to ink to Optimus WIKI page, which we don't have yet. Insights? @barkasn @kbergin @TimothyTickle

**Review by samanehsan — APPROVED** [review]

**Review by infispiel — APPROVED** [review]

**Review by kishorikonwar — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -17,7 +20,7 @@ For now, use `git clone git@github.com:HumanCellAtlas/skylab.git` and run the pi
 
 Tasks on Cromwell may be run on what are known as "preemptible" machines to reduce costs by a significant amount. The catch with preemptible machines is that they may be "preempted" at any given moment--as in, google may shut down the task to re-use the resources.
 
-Many tasks are set to automatically be `preemptible = 3`, aka they will be run on preemptible instances for up to 3 instances of preemption, after which it will be run on a non-preemptible machine. The Optimus pipeline in particular also has a `preemptible` input that overrides the default preemptible parameter on all tasks run through optimus. This option may be set to 0 to run the entire workflow without using preemptible machines.
+Many tasks are set to automatically be `preemptible = 3`, aka they will be run on preemptible instances for up to 3 instances of preemption, after which it will be run on a non-preemptible machine. This option may be set to 0 to run the task without using preemptible machines for those long-running tasks.
```
Comment (infispiel) [inline README.md]: Maybe worth adding something like --

> This option may be set to 0 by passing a task-level input to the workflow (i.e. `Optimus.StarAlign.preemptible`), causing the task to be run without using preemptible machines. This option is especially useful for long-running tasks, which have a chance to take a very long time to run due to being preempted multiple times.

Comment (kbergin) [reply README.md]: +1 ^

File: pipelines/optimus/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Optimus Pipeline
+
+The Optimus pipeline is a pipeline for processing 3' single-cell expression data generated with the 10X Genomic V2 assay, developed by the Data Coordination Platform (DCP) of the Human Cell Atlas (HCA) Project. 
+
+Optimus is a alignment and transcriptome quantification pipeline. Optimus corrects Cell Barcodes (CBs) and Unique Molecular Identifiers (UMIs), aligns reads to the genome, generates an expression count matrix in a UMI-aware manner, detects empty droplets, calculates summary statistics for genes and cells, and returns outputs in BAM and Zarr file formats. Special care is taken to keep  all reads that may be useful to the downstream user, such as unaligned reads or reads with uncorrectable barcodes. This design provides flexibility to the downstream user and allows for alternative filtering or leveraging the data for novel methodological development.
```
Comment (kbergin) [inline pipelines/optimus/README.md]: Could we add the diagram of the pipeline to this readme? Thanks!

**Review by kbergin — APPROVED** [review]
Just a couple nits. YAY!!!!!

File: pipelines/optimus/README.md (hunk 1 above)
Comment (barkasn) [reply pipelines/optimus/README.md]: The report is completely open to the public like all HCA work. Link is here: https://docs.google.com/document/d/158ba_xQM9AYyu8VcLWsIvSoEYps6PQhgddTr9H0BFmY/edit

File: pipelines/optimus/README.md (hunk 3 above)
Comment (rexwangcc) [reply pipelines/optimus/README.md]: Adding!

Comment (rexwangcc) [reply pipelines/optimus/README.md]: Done!

File: README.md (hunk 2 above)
Comment (rexwangcc) [reply README.md]: Done

File: pipelines/optimus/README.md (hunk 1 above)
Comment (rexwangcc) [reply pipelines/optimus/README.md]: @barkasn OK, then we could link to it. Done.

**Review by pshapiro4broad — APPROVED** [review]

**Pull request merged**